sample-lnd.conf

```diff
@@ -963,6 +963,37 @@
 ; Instructs lnd to encrypt the private key using the wallet's seed.
 ; tor.encryptkey=false
 
+[logging]
+
+; Disable logging to stdout and stderror.
+; Default:
+;   logging.console.disable=false
+; Example:
+;   logging.console.disable=true
```

Comment:
nit: Do we really need an example for boolean values? I think the more compact form with just the default value is probably enough.

Reply:
ah - missed this comment. Will address tomorrow

Reply:
fixed